Source/WebCore/platform/graphics/MediaPlayer.cpp

```diff
@@ -923,6 +923,11 @@ FloatSize MediaPlayer::videoLayerSize() const
     return client().mediaPlayerVideoLayerSize();
 }
 
+void MediaPlayer::setVideoLayerSize(const FloatSize& size)
+{
+    client().mediaPlayerDidSetVideoLayerSize(size);
+}
+
```

Comment:
`MediaPlayer` is a weird class that both passes information down to MediaPlayerPrivate, up to MediaPlayerClient, and also does some amount of caching. That said, the naming of this method seems wrong.

Maybe this could be `MediaPlayer::videoLayerSizeDidChange(const FloatSize& size)`?